public/app/plugins/panel/stat/types.ts

```diff
@@ -65,6 +72,34 @@ export function addStandardDataReduceOptions(
     showIf: currentConfig => currentConfig.reduceOptions.values === false,
   });
 
+  if (includeFieldMatcher) {
+    builder.addSelect({
+      path: 'reduceOptions.fields',
+      name: 'Fields',
+      description: 'Select the fields that should be included in the panel',
+      settings: {
+        allowCustomValue: true,
+        options: [],
+        getOptions: (context: FieldOverrideContext) => {
+          const options = [
+            { value: '', label: 'Numeric Fields' },
+            { value: '/.*/', label: 'All Fields' },
+          ];
+          if (context && context.data) {
+            for (const frame of context.data) {
+              for (const field of frame.fields) {
+                const name = getFieldDisplayName(field, frame, context.data);
+                options.push({ value: name, label: name });
```

Comment:
since it's using the regex matcher should it regex escape values here?

Reply:
yup -- good catch